tools/tpm2_import.c

```diff
@@ -982,6 +982,42 @@ static bool load_key(
     return true;
 }
 
+/**
+ * Check all options and report as many errors as possible via LOG_ERR.
+ * @return
+ *  0 on success, 1 on failure.
```

Comment:
Might be nice to return -1 instead of 1 for errors in this function, that way the usage information will be shown?

Reply:
Yeah that's a good idea.